db/migrate/20220428133704_rename_type_to_show_type.rb

```diff
@@ -0,0 +1,5 @@
+class RenameTypeToShowType < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :shows, :type, :show_type
```

Comment:
Ignore comment above now that I have seen this. Just wondering why you didn't amend the original migration that creates the table?